dataflow/src/main/java/org/checkerframework/dataflow/analysis/FlowExpressions.java

```diff
@@ -1057,6 +1065,190 @@ public String toString() {
         }
     }
 
+    /** FlowExpression.Receiver for binary operations. */
+    public static class BinaryOperation extends Receiver {
+
+        /** The binary operation kind. */
+        protected final Kind operationKind;
+        /** Receiver of the left operand. */
+        protected final Receiver left;
+        /** Receiver of the right operand. */
+        protected final Receiver right;
+
+        /**
```

Comment:
You will need summaries in all javadoc, since we now use an errorprone version that enforces that. So just add those for all your methods.